pkg/config/user_config_validation.go

```diff
@@ -2,12 +2,18 @@ package config
 
 import (
 	"fmt"
+	"regexp"
 	"slices"
 	"strings"
 )
 
 func (config *UserConfig) Validate() error {
-	if err := validateEnum("gui.commitAuthorFormat", config.Gui.CommitAuthorFormat,
+	if strings.HasPrefix(string(config.Gui.CommitAuthorFormat), "truncateTo:") {
+		regex := regexp.MustCompile(`truncateTo:\d+(,\d+)?$`)
```

Comment:
We have this regex (or very similar ones, that is) in two places, so it might be worth extracting it somehow. I was too lazy to think about exactly how...